cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_bulletin_config.rs

```diff
@@ -0,0 +1,267 @@
+};
+use bp_messages::LaneId;
+use bridge_runtime_common::{
+	messages,
+	messages::{
+		source::{FromBridgedChainMessagesDeliveryProof, TargetHeaderChainAdapter},
+		target::{FromBridgedChainMessagesProof, SourceHeaderChainAdapter},
+		MessageBridge, UnderlyingChainProvider,
+	},
+	messages_xcm_extension::{
+		SenderAndLane, XcmAsPlainPayload, XcmBlobHauler, XcmBlobHaulerAdapter,
+		XcmBlobMessageDispatch,
+	},
+	refund_relayer_extension::{
+		ActualFeeRefund, RefundBridgedGrandpaMessages, RefundSignedExtensionAdapter,
+		RefundableMessagesLane,
+	},
+};
+
+use frame_support::{parameter_types, traits::PalletInfoAccess};
+use sp_runtime::RuntimeDebug;
+use xcm::{
+	latest::prelude::*,
+	prelude::{InteriorMultiLocation, NetworkId},
+};
+use xcm_builder::{BridgeBlobDispatcher, HaulBlobExporter};
+
+parameter_types! {
+	/// Maximal number of entries in the unrewarded relayers vector at the Rococo Bridge Hub. It matches the
+	/// maximal number of unrewarded relayers that the single confirmation transaction at Rococo Bulletin Chain
+	/// may process.
+	pub const MaxUnrewardedRelayerEntriesAtInboundLane: bp_messages::MessageNonce =
+		bp_polkadot_bulletin::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+	/// Maximal number of unconfirmed messages at the Rococo Bridge Hub. It matches the maximal number of
+	/// unconfirmed messages that the single confirmation transaction at Rococo Bulletin Chain may process.
+	pub const MaxUnconfirmedMessagesAtInboundLane: bp_messages::MessageNonce =
+		bp_polkadot_bulletin::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+	/// Bridge specific chain (network) identifier of the Rococo Bulletin Chain.
+	pub const RococoBulletinChainId: bp_runtime::ChainId = bp_runtime::POLKADOT_BULLETIN_CHAIN_ID;
+	/// Interior location (relative to this runtime) of the with-RococoBulletin messages pallet.
+	pub BridgeRococoToRococoBulletinMessagesPalletInstance: InteriorMultiLocation = X1(
+		PalletInstance(<BridgeRococoBulletinMessages as PalletInfoAccess>::index() as u8)
+	);
+	/// Rococo Bulletin Network identifier.
+	pub RococoBulletinGlobalConsensusNetwork: NetworkId = NetworkId::PolkadotBulletin;
+	/// All active lanes that the current bridge supports.
+	pub ActiveOutboundLanesToRococoBulletin: &'static [bp_messages::LaneId]
+		= &[XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN];
+	/// Lane identifier, used to connect Rococo People and Rococo Bulletin chain.
+	pub const RococoPeopleToRococoBulletinMessagesLane: bp_messages::LaneId
+		= XCM_LANE_FOR_ROCOCO_PEOPLE_TO_ROCOCO_BULLETIN;
+	/// Priority boost that the registered relayer receives for every additional message in the message
+	/// delivery transaction.
+	///
+	/// It is determined semi-automatically - see `FEE_BOOST_PER_MESSAGE` constant to get the
+	/// meaning of this value.
+	pub PriorityBoostPerMessage: u64 = 182_044_444_444_444;
+
+	/// Identifier of the sibling Rococo People parachain.
+	pub RococoPeopleParaId: cumulus_primitives_core::ParaId = 1004.into();
```

Comment:
also better to use rococo's system parachain constants mentioned in previous comment

Reply:
done